src/benchmark.jl

```diff
@@ -25,7 +25,7 @@ macro benchmark(core)
                 $(esc(core)),
                 nothing
             )
-            Benchmarks.execute($name)
+            Benchmarks.BenchmarkResults(Benchmarks.execute($name))
```

Comment:
I don't think we should auto-generate summary statistics for all benchmarks. This makes sense now because the statistical analysis is fairly cheap. If we switch to something like bootstrapping by default, it's less clear to me that we should force people to pay the cost for an analysis they won't use.